### PR TITLE
Logo animation

### PR DESCRIPTION
**Pull request by tcoots88**

### CODE REVIEW

File: Logo/parse-life animation.html [hunk 1]
```diff
@@ -0,0 +1,98 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <link rel="stylesheet" href="parse-life animation.css">
```
Comment (phongvdoan) [inline Logo/parse-life animation.html]: Put in index.html

File: Logo/parse-life animation.html [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+  <link href="https://fonts.googleapis.com/css?family=B612+Mono|Cutive+Mono|Gruppo|Life+Savers|Oxygen+Mono|PT+Mono|Press+Start+2P|Turret+Road&display=swap" rel="stylesheet">
```
Comment (phongvdoan) [inline Logo/parse-life animation.html]: place in index.html

File: Logo/parse-life animation.html [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+      <p class="text" id="Press">
+          Parse<!--This comment is required to remove whitespace created by inline-block elements.
+       --><span>
+              <span class="stack bouncing">.</span>
+              <!-- <span class="stack">ı</span> -->
+          </span><!--This comment is required to remove whitespace created by inline-block elements.
+       -->life
+      </p>
+      <br>
+      <br>
```
Comment (phongvdoan) [inline Logo/parse-life animation.html]: place in index.html

**Review by phongvdoan — CHANGES_REQUESTED** [review]

**Comment by tcoots88** [comment]
Added into index.html

**Review by phongvdoan — CHANGES_REQUESTED** [review]
place contents of parse-life animation .css into style/style.css, because the lower css will override the other

remove the link for parse-life animation.css inside index.html

**Review by phongvdoan — APPROVED** [review]

**Pull request merged**